proto/babylon/btcstaking/v1/query.proto

```diff
@@ -315,3 +323,26 @@ message QueryLargestBtcReOrgResponse {
   // RollbackTo is the BTC block header which we rollback to
   babylon.btclightclient.v1.BTCHeaderInfoResponse rollback_to = 3;
 }
+
+
+// QueryParamsVersionsRequest is the request type for the
+// Query/ParamsVersions RPC method.
+message QueryParamsVersionsRequest {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
+}
+
+// QueryParamsVersionsResponse stores all the params with versions.
+message QueryParamsVersionsResponse {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // params holds all the params with version from this module.
+  repeated StoredParams params = 1 [ (gogoproto.nullable) = false ];
```

Comment:
ideally we would not expose type used for db storage, only `Params` struct as:
- every other params query contains only `Params` struct
- now every change to `StoredParams` is both api and state breaking
Though given this is backport, it is not blocking 👍

Reply:
damm, you are correct 
I will make a PR to fix in main and backport to release

Reply:
fix at #651